### PR TITLE
feat: #9 remove custom report template and use lsd-core html renderer

### PR DESCRIPTION
**Pull request by nickmcdowall**
This will help make sure the styling and javascript remain in sync and should therefore require less maintenance

### CODE REVIEW

File: service/src/componentTest/kotlin/com/lsdconsulting/generatorui/LsdDistributedGeneratorUiApiTest.kt [hunk 1]
```diff
@@ -45,7 +45,7 @@ class LsdGeneratorUiApiTest(
         val url = "/lsd/${interceptedInteraction.traceId}"
         val result = testRestTemplate.exchange(url, GET, HttpEntity<Nothing>(headers), String::class.java)
 
-        assertThat(result.body!!, containsSubstring("Diagram for traceIds: ${interceptedInteraction.traceId}"))
+        assertThat(result.body!!, containsSubstring("Diagram for traceIds"))
```
Comment (lukasz-gryzbon) [inline service/src/componentTest/kotlin/com/lsdconsulting/generatorui/LsdDistributedGeneratorUiApiTest.kt:48]: Maybe we should still assert if the `traceId` is rendered?

File: service/src/main/kotlin/com/lsdconsulting/generatorui/service/LsdGenerator.kt [hunk 2]
```diff
@@ -1,23 +1,77 @@
...
         val eventContainer = interactionGenerator.generate(traceIdToColourMap)
         val participants = participantListGenerator.generateParticipants(eventContainer.events)
-        val title = "Diagram for traceIds: ${traceIds.asList().joinToString()}"
-        val scenario = scenarioBuilder.build(title = title, events = eventContainer.events, traceIds = traceIds.asList(), participants = participants.toList())
-        return htmlReportRenderer.render(scenario = scenario, startTime = eventContainer.startTime, finishTime = eventContainer.finishTime)
+
+        return htmlReportRenderer.render(
+            Report(
+                title = "Event Report",
+                status = "success",
```
Comment (lukasz-gryzbon) [inline service/src/main/kotlin/com/lsdconsulting/generatorui/service/LsdGenerator.kt:35]: What is the meaning of this status in the context of this project?

**Review by lukasz-gryzbon — APPROVED** [review]
Good work! Thanks!

Just a couple of small small things.

File: service/src/componentTest/kotlin/com/lsdconsulting/generatorui/LsdDistributedGeneratorUiApiTest.kt (hunk 1 above)
Comment (nickmcdowall) [reply service/src/componentTest/kotlin/com/lsdconsulting/generatorui/LsdDistributedGeneratorUiApiTest.kt:48]: hey, happy to add additional check for this (I did notice that the adjacent approval test already covers this and also proves the html is returned so maybe that particular test is unnecessary?).

File: service/src/main/kotlin/com/lsdconsulting/generatorui/service/LsdGenerator.kt (hunk 2 above)
Comment (nickmcdowall) [reply service/src/main/kotlin/com/lsdconsulting/generatorui/service/LsdGenerator.kt:35]: Good question - it's not actually required in this context so I can remove! 😄

**Pull request merged**